### PR TITLE
[1LP][RFR] 5.9 paginator enhancement

### PR DESCRIPTION
**Pull request by izapolsk**
devs added min_item/max_item data to JS Paginator API. So, we can start using it in our tests.

### CODE REVIEW

**Comment by izapolsk** [comment]
Please ignore 5.9 errors. Lastest JS API changes aren't merged into 5.9.0.5 for some reason

File: cfme/tests/automate/test_customization_paginator.py [hunk 1]
```diff
@@ -61,27 +61,26 @@ def test_paginator(some_dialogs, soft_assert, appliance):
         * During the cycling, assert the numbers displayed in the paginator make sense
         * During the cycling, assert the paginator does not get stuck.
     """
-    navigate_to(DialogCollection(appliance), 'All')
-    from cfme.web_ui import paginator
-    paginator.results_per_page(50)
-    paginator.results_per_page(5)
+    view = navigate_to(DialogCollection(appliance), 'All')
```
Comment (mshriver) [inline cfme/tests/automate/test_customization_paginator.py]: **Question**
Why not `appliance.collections.service_dialogs`?

Comment (izapolsk) [reply cfme/tests/automate/test_customization_paginator.py]: @mshriver, Mike I don't want to change someone's tests.  I just add my enhancement.

**Comment by mshriver** [comment]
Paying attention to upstream PRT results in 17676, as 5.9.0.5 doesn't have the JS methods yet.

**Pull request merged**